.github/workflows/acceptance.yml

```diff
@@ -6,6 +6,33 @@ on:
   pull_request:
 
 jobs:
+    getsentry:
+      if: ${{ github.ref != 'refs/heads/master' && !contains(github.event.commits[0].message, '#test-getsentry') }}
```

Comment:
Sorry, should have caught the `!contains`. In my defense no syntax highlighting here makes it much harder.